### ISSUE TITLE
Increased font size breaks layout

### ISSUE THREAD

**Issue by jidanni** [mdn/yari#11499]
### MDN URL

Many pages, the entire site in fact,

### What information was incorrect, unhelpful, or incomplete?

Here is what the website looks like on a Chromebook,

![xx](https://github.com/user-attachments/assets/e6571586-613f-47c5-95e1-4b711af09619)

### What did you expect to see?

I expected to see more of the page,
but all we get to see is tiny box B.
That's the only part of the page that scrolls down.

On other websites, we could be finished reading a document before bedtime.

But here, due to needing baby-steps scrolling, it will take longer.

One also notices:

1. Totally wasted area A.
2. Additional wasted area C.

I am sorry, but:
- Users give up scrolling tiny window B, and find some other website to read that is not so painful.
- Users wonder how could a website that discusses website design be so poorly designed. Perhaps all the articles on it are not to be trusted, if websites like this are the result.

Stackoverflow on the other hand, allows the user to scroll the entire M area:

![ss1](https://github.com/user-attachments/assets/5a9bf30a-3a5d-4812-b816-ae8c5c419dbb)

**Comment by caugner** [comment]
Hi @jidanni, I have some questions:

1. Can you confirm that you're using Chrome and have set the "Font size" setting under "Appearance" to "Large" or "Very large"?
2. What is your screen resolution?
3. Can you share links to MDN pages where you experience issue?

I can reproduce the wrapping top menu (A), but I cannot reproduce the white area on the right (C). It is possible that the page (which was meanwhile replaced by [this one](https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Styling_basics/Backgrounds_and_borders)) had a very short table of contents that was hidden behind the wrapping top menu.

**Comment by jidanni** [comment]
Oops, forgot to put some URLs in my report.
But e.g.,
https://developer.mozilla.org/en-US/blog/introduction-to-web-sustainability/
any one I look at looks fine now. So fine that I'll just close this bug now fast!
